### PR TITLE
Fix lora layers

### PR DESCRIPTION
**Pull request by richardsliu**
# Description

Fix broken lora layers test

FIXES: b/459571963

# Tests

Please describe how you tested this change, and include any instructions and/or
commands to reproduce.

# Checklist

Before submitting this PR, please make sure:
- I have performed a self-review of my code.
- I have necessary comments in my code, particularly in hard-to-understand areas.
- I have made or will make corresponding changes to any relevant documentation.

### CODE REVIEW

**Comment by richardsliu** [comment]
CI: https://buildkite.com/tpu-commons/tpu-inference-ci/builds/5186#_

**Comment by vanbasten23** [comment]
Thanks for the fix.

The error happens at `tensor.detach().cpu()` because the tensor is torchax.tensor.Tensor'.
